style.css

```diff
@@ -47,38 +47,6 @@
     border-radius: 2px;
 }
 
-#blast1 {
-    background: red;
+.blast {
```

Comment:
Usually when I have a lot of duplicated CSS, I ask myself if I could add both a class and an ID to the element